webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      filename: 'index.html',
+      template: './src/index.html',
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        exclude: /node_modules/, // Exclude node_modules
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
```

Comment:
- [ ] Nice work using Webpack for the implementation of the project :+1:  However Webpack **dev server** is not set up in your Webpack configuration. Kindly make sure Webpack **dev server** is set up and running with no errors